services/order.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
```

Comment:
Better split this params on separate lines for readability + date missing Optional type hint